lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -34,6 +35,47 @@ import 'window.dart';
 ui.VoidCallback? scheduleFrameCallback;
 
 typedef _KeyDataResponseCallback = void Function(bool handled);
+typedef HighContrastListener = void Function(bool);
+
+
+/// Determines if high contrast is enabled using media query 'forced-colors: active' for windows
+/// TODO(negarb): Add support for web on Mac/iOS/andriod
+class HighContrastSupport {
+  static HighContrastSupport instance = HighContrastSupport();
+  static const String _highContrastMediaQueryString = '(forced-colors: active)';
+
+  final List<HighContrastListener> listeners = <HighContrastListener>[];
+
+  /// Reference to css media query that indicates whether high contrast is on.
+  final html.MediaQueryList _highContrastMediaQuery =
+      html.window.matchMedia(_highContrastMediaQueryString);
+
+  bool isHighContrastEnabled() {
```

Comment:
It would probably be more idiomatic as a getter (i.e. `bool get isHighContrastEnabled { ... }`).